Kickstarter-iOS/DataSources/PledgeDataSource.swift

```diff
@@ -23,8 +23,8 @@ final class PledgeDataSource: ValueCellDataSource {
     )
 
     self.appendRow(
-      value: "Your shipping location",
-      cellClass: PledgeRowCell.self,
+      value: (location: "British Indian Ocean Territory", currency: "$", rate: 7.50),
```

Comment:
These values are currently hard coded in order for this PR to be smaller. I'll be passing them from the `VM` in the next PR.